packages/fs-detectors/src/detectors/filesystem.ts

```diff
@@ -96,12 +96,23 @@ export abstract class DetectorFilesystem {
   /**
    * Writes a file to the filesystem cache.
    * @param name the name of the file to write
-   * @param content The content of the file
+   * @param content contents of the file
+   * @param options.exists optional. undefined or true (default) if the files exists, false otherwise
    */
-  public writeFile(name: string, content?: string): void {
-    if (content)
-      this.readFileCache.set(name, Promise.resolve(Buffer.from(content)));
-    this.fileCache.set(name, Promise.resolve(true));
-    this.pathCache.set(name, Promise.resolve(true));
+  public writeFile(
+    name: string,
+    content: string,
```

Comment:
Now `content` is not an optional parameter. I am wondering if that breaks any existing uses of `writeFile` 🤔